src/sass/style.scss

```diff
@@ -13,9 +13,9 @@ $button-hover: $active-bg; //#31bbff;
 
 // fonts
 $fa-font-path: "fonts";
-@import "../../node_modules/font-awesome/scss/font-awesome.scss";
```

Comment:
This doesn't work on my local test

Reply:
How are you testing (inside a project, inside CE)?

Reply:
I just found out something that could solve our problems: https://github.com/stackcss/sheetify#use-npm-packages